velox/substrait/SubstraitUtils.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "substrait/algebra.pb.h"
+#include "substrait/capabilities.pb.h"
+#include "substrait/extensions/extensions.pb.h"
+#include "substrait/function.pb.h"
+#include "substrait/parameterized_types.pb.h"
+#include "substrait/plan.pb.h"
+#include "substrait/type.pb.h"
+#include "substrait/type_expressions.pb.h"
+
+namespace facebook::velox::substrait {
+
+/// This class contains some common funcitons used to parse Substrait
+/// components, and convert them into recognizable representations.
+class SubstraitParser {
+ public:
+  /// Used to store the type name and nullability.
+  struct SubstraitType {
+    std::string type;
+    bool nullable;
+    SubstraitType(const std::string& subType, const bool& subNullable) {
```

Comment:
use constructor intializers? Also, you might not need to explicitly specify this constructor - it's available by default (`SubstraitType{a, b} type`).

Reply:
Fixed.